externs/ngeox.js

```diff
@@ -258,6 +258,43 @@ ngeox.format.FeatureHashOptions.prototype.properties;
  */
 ngeox.interaction;
 
+
+/**
+ * @typedef {{
+ *     pixelTolerance: (number|undefined),
+ *     style: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *     features: ol.Collection.<ol.Feature>,
+ *     wrapX: (boolean|undefined)}}
+ * @api
+ */
+ngeox.interaction.ModifyOptions;
```

Comment:
These are not specific to the circle modification interation though, it's actually just about the same as the `olx.interaction.ModifyOptions`, and I think all modify interactions which inherit from ol's modify interactions will need these options.

(Actually I don't know why just using `olx.interaction.ModifyOptions` wasn't working, since it's really all I needed)